aiidalab/utils.py

```diff
@@ -147,23 +148,43 @@ def wrapped(instance, *args, **kwargs):
 class Package:
     """Helper class to check whether a given package fulfills a requirement."""
 
-    def __init__(self, name, version):
-        self.name = name
+    def __init__(self, name: str, version: str = None):
```

Comment:
Why was this None default added? Can you explain the use case?

Reply:
This is used in `_CORE_PACKAGES = [Package("aiida-core"), Package("jupyter-client")]`, as you suggest, I use the Package instance and we shouldn't specify version to them.

Reply:
Oh, I see, got it. Thanks!